ui/admin/app/components/form/host-set/azure/index.hbs

```diff
@@ -12,30 +12,31 @@
   as |form|
 >
 
-  <form.input
-    @name='name'
-    @type='name'
+  <Hds::Form::TextInput::Field
+    name='name'
+    @isOptional={{true}}
     @value={{@model.name}}
```

Comment:
None blocking comment: I will suggest the attribute `@value` to go before `@isOptional`

Reply:
@calcaide totally agree, quick commit with changes incoming